digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/ProjectDAO.scala

```diff
@@ -173,6 +173,19 @@ object ProjectDAO {
     Q.queryNA[String](query).firstOption
   }
 
+
+  def isNewRoadPartLinkUsed(roadNumber: Long, roadPartNumber: Long, projectId: Long) = {
+    val query =
+      s"""
+		select prol.id
+         from project_link prol
+         join lrm_position pos on prol.lrm_position_id = pos.id
```

Comment:
Possibly to make sure that the Project_Link has a LRM_Possition connection therefore being a step closer to prove that it is correctly created (therefore being able to be used)?